src/Views/EncryptView.vala

```diff
@@ -52,7 +52,7 @@ public class EncryptView : AbstractInstallerView {
         };
         title_label.get_style_context ().add_class (Granite.STYLE_CLASS_H2_LABEL);
 
-        var details_label = new Gtk.Label (_("Encrypt this device's drive if required for added protection, but be sure you understand:")) {
+        var details_label = new Gtk.Label (_("Encrypting this device's drive ensures the safety of your information. We recommend enabling drive encryption which means:")) {
```

Comment:
What about this instead:

```suggestion
        var details_label = new Gtk.Label (_("Encrypting this device's drive improves the safety of your information. We recommend enabling drive encryption which means:")) {
```

Reply:
It still feels weird to say we recommend it, but then the default is not to enable it (for good reason, namely the localization, input method, and sync issues). I'm not totally against rewording this, but again, the original copy was carefully iterated several times for very specific reasons.